src/App.js

```diff
@@ -1,13 +1,27 @@
 import React from "react";
-
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer";
 import Bookings from "./Bookings";
+import SearchResults from "./SearchResults";
 import "./App.css";
+import Heading from "./Heading";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      {/* <header className="App-header">CYF Hotel DJMS</header> */}
+      <Heading />
+
       <Bookings />
+
+      <TouristInfoCards />
+      <Footer
+        addressDetail={[
+          "123 Fake Street, London, E1 4UD",
+          "hello@fakehotel.com",
+          "0123 456789",
+        ]}
+      />
```

Comment:
Although this code works, on a readability note this bit could be improved

Perhaps renaming `addressDetail `to something like `contactDetails`


Or something like

```
<Footer
        address="123 Fake Street, London, E1 4UD"
        email="hello@fakehotel.com"
        phoneNumber="0123 456789"
 />
```